package.json

```diff
@@ -129,6 +129,7 @@
   "readmeFilename": "README.md",
   "license": "Apache-2.0",
   "dependencies": {
+    "@babel/runtime": "^7.17.7",
```

Comment:
This should not be part of the dependencies, but instead dev-dependencies, as its packaged and included via webpack. Please update.